src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double base;
+    private double height;
+
+    public RightTriangle(String color, double base, double height) {
+        super(color);
+        this.base = base;
+        this.height = height;
+    }
+
+    public RightTriangle(String color) {
+        super(color);
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * base * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Фігура: прямокутний трикутник, площа: " + getArea()
+               + " кв. одиниць, перший катет: " + base + " одиниць, другий катет: "
+                + height + " одиниць, колір: " + getColor());
```

Comment:
Use only English in messages/code. The message in the `draw()` method should be in English to make the code universal and consistent.